pages/docs/reference/java-to-kotlin-interop.md

```diff
@@ -333,24 +333,27 @@ making its getter and setter methods static members in that object or the class
 > Default methods are available only for targets JVM 1.8 and above.
 {:.note}
 
-> The `@JvmDefault` annotation is experimental in Kotlin 1.3. Its name and behavior may change, leading to future incompatibility.
-{:.note}
-
 Starting from JDK 1.8, interfaces in Java can contain [default methods](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html).
-You can declare a non-abstract member of a Kotlin interface as default for the Java classes implementing it.
-To make a member default, mark it with the [`@JvmDefault`](/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/index.html) annotation.
+To make all non-abstract members of Kotlin interfaces default for the Java classes implementing them, compile the Kotlin 
+code with the `-Xjvm-default=all` compiler option.
+
 Here is an example of a Kotlin interface with a default method:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
+// compile with -Xjvm-default=all
+
 interface Robot {
-    @JvmDefault fun move() { println("~walking~") }
+    fun move() { println("~walking~") }  // will be default in the Java interface
     fun speak(): Unit
 }
 ```
 </div>
 
+Compiling with `-Xjvm-default=all` has the same effect as annotating all non-abstract methods of interfaces with
```

Comment:
It's not totatlly true because of delegation by difference

Reply:
Since we're gonna deprecate `@JvmDefault` in 1.4.20, maybe we can skip this detail? Just add `In general` or `In most cases`.
Otherwise, we'll have to keep the entire section about delegation with `@JvmDefault` which gets deprecated really soon.

Reply:
+1 for "in general" / "in most cases". Also, maybe clarify in a footnote, if footnotes are supported? :)